server/graphql/schema.ts

```diff
@@ -105,6 +106,7 @@ const typeDefs = gql`
         id: ID
         clientId: String
         fullName: String
+        deleted: Boolean
```

Comment:
why are we adding deleted? is this related to the filtering logic?

Reply:
Will address this in a new PR (including changes discussed on Slack)